engine/requirements.txt

```diff
@@ -248,8 +248,6 @@ jsonschema-specifications==2023.12.1
     # via jsonschema
 kombu==5.3.5
     # via celery
-lxml==4.9.2
-    # via -r requirements.in
```

Comment:
this doesn't seem to be used anywhere? (this version seemed to be causing some issues when running `pip install -r requirements.txt` on a fresh virtualenv of Python 3.12.4.. so trying to see if we don't actually need it?)